content/sensu-go/5.0/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Sensu Go"
-description: "Sensu is the flexible monitoring event pipeline built to reduce operator burden and meet the challenges of monitoring hybrid-cloud and ephemeral infrastructures. Read the docs to learn more about Sensu."
+description: "Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflow and gain deep visibility into their infrastructure, applications, and operations. Read the docs to learn more about Sensu."
```

Comment:
make "workflows" plural, please

Reply:
"Monitor your entire infrastructure, from Kubernetes to bare metal. The Sensu monitoring event pipeline empowers businesses to automate their monitoring workflows and gain deep visibility into their infrastructure, applications, and operations. Try Sensu today."